containers/Dockerfile

```diff
@@ -42,10 +42,12 @@ ENV PROJECT_DIR=/projects \
 # install TA-Lib and other prerequisites
 #
 
-RUN mkdir ${PROJECT_DIR} \
-    && apt-get -y update \
-    && apt-get -y install libfreetype6-dev libpng-dev libopenblas-dev liblapack-dev gfortran \
-    && curl -L https://downloads.sourceforge.net/project/ta-lib/ta-lib/0.4.0/ta-lib-0.4.0-src.tar.gz | tar xvz
+RUN mkdir ${PROJECT_DIR}
```

Comment:
nit: `mkdir -p ${PROJECT_DIR}`

Without knowledge of what these variables point to we do not know if these paths are more than one level deep.